test/helper.rb

```diff
@@ -3,11 +3,13 @@
 require 'rubygems/mock_gem_ui'
 require 'json/jwt'
 
-require_relative 'support/webmock_helper'
-require_relative 'support/url_helper'
-require_relative 'support/sigstore_auth_helper'
-require_relative 'support/fulcio_helper'
-require_relative 'support/rekor_helper'
+require 'rubygems/sigstore'
```

Comment:
I'm not too sure about this one.  @doodzik does it make sense to effectively preload much of the plugin's code up front in our test helper?